### PR TITLE
Fig4 discussion rewrite

### PR DESCRIPTION
**Pull request by Hugemiler**
Additions to discussion pertinent to figures 3 and 4

### CODE REVIEW

**Comment by Hugemiler** [comment]
@kescobo partial reviews welcome on every new paragraph added. I'm gonna proceed to add another paragraph for the subscales and then tweak numbers and species on the already-existing brain-cog paragraphs.

File: manuscript/main.tex [hunk 1]
```diff
@@ -852,6 +845,31 @@ \section*{Discussion}
...
 targets for further mechanistic studies.
 
+While we did not identify any statistical associations between individual species
+and cognitive performance in our youngest subset (0 to 6 months),
+it may be because brain developmental trajectories begin \textit{in utero},
+and microbial influences (that can only begin at birth)
+take additional time to manifest.
+A prediction experiment aligned to that hypothesis,
+targeting future cognitive assessment scores using metagenomes acquired prior to 1 year of age,
+revealed a list of most predictive taxa for that task.
```
Comment (kescobo) [inline manuscript/main.tex:855]:
```suggestion
In line with this hypothesis, RF models that were able to learn associations
between early-life microbiomes and future cognitive function scores (Figure 3).
```

File: manuscript/main.tex [hunk 2]
```diff
@@ -852,6 +845,31 @@ \section*{Discussion}
...
+The top-ranked species on that list include several
```
Comment (kescobo) [inline manuscript/main.tex:856]:
```suggestion
The top-ranked species for these models included several 
```

File: manuscript/main.tex [hunk 3]
```diff
@@ -852,6 +845,31 @@ \section*{Discussion}
...
+and microbial influences (that can only begin at birth)
+take additional time to manifest.
+A prediction experiment aligned to that hypothesis,
+targeting future cognitive assessment scores using metagenomes acquired prior to 1 year of age,
+revealed a list of most predictive taxa for that task.
+The top-ranked species on that list include several 
+taxa that were important for predicting concurrent cognitive assessment scores after 18 months, notably
+pathobiontic \textit{Enterobacteriaceae} such as
+\textit{Escherichia} and \textit{Klebsiella}. 
+Representatives of this genus are known to be early and long-term persistent
```
Comment (kescobo) [inline manuscript/main.tex:860]:
```suggestion
Representatives of these genera are known to be early and long-term persistent
```

File: manuscript/main.tex [hunk 4]
```diff
@@ -852,6 +845,31 @@ \section*{Discussion}
...
+colonizers of infant intestine that can, in some settings,
+promote inflammatory response \cite{popeEnterobacEarlyGut2019}.
+Parallel to those species, several tradinionally health-promoting and inflammation-mudulating representatives
```
Comment (kescobo) [inline manuscript/main.tex:863]:
```suggestion
By contrast, several traditionally inflammation-reducing representatives
```

File: manuscript/main.tex [hunk 5]
```diff
@@ -852,6 +845,31 @@ \section*{Discussion}
...
+of the \textit{Bifidobacterium} genus,
+from the early HMO-metabolizer \textit{B. longum} to the
+later complex carbohydrate relying \textit{B. pseudocatenulatum} \cite{chungBifidoPsudocatenulatum2021},
+commonly associated with weaning to solid food in early age,
+have also shown high relative importance.
```
Comment (kescobo) [inline manuscript/main.tex:868]:
```suggestion
also showed high relative importance.
```

File: manuscript/main.tex [hunk 6]
```diff
@@ -852,6 +845,31 @@ \section*{Discussion}
...
+The involvement of most important taxa in different aspects of inflammatory response
+resonate with previous findings that associate chronic inflammation with
+changes to neural responses in some types of stimula, notably visual\cite{xieChronicInfNeuro2019}.
```
Comment (kescobo) [inline manuscript/main.tex:871]:
```suggestion
changes to neural responses in some types of stimulae, notably visual\cite{xieChronicInfNeuro2019}.
```

File: manuscript/main.tex [hunk 7]
```diff
@@ -881,6 +899,24 @@ \section*{Discussion}
 it has been shown to have a divergent neuromorphology
 \cite{hoogmanSubcorticalBrainVolume2017}.
 
+Our ability to probe individual cognitive subscales provided
+results that also endorse previous findings and lay insight to
```
Comment (kescobo) [inline manuscript/main.tex:903]:
```suggestion
results that also endorse previous findings and provide insight to
```

File: manuscript/main.tex [hunk 8]
```diff
@@ -881,6 +899,24 @@ \section*{Discussion}
...
+potential gut-microbial connections between cognition and neuroanatomy.
+For example, a longitudinal neurocognitive development study pointed at bilateral
+associations between the thalamus and the EL component of the Mullen scales.
+Linking to this observation, we found previously-mentioned taxa such as
```
Comment (kescobo) [inline manuscript/main.tex:907]:
```suggestion
In line with this finding, we found that taxa such as 
```

File: manuscript/main.tex [hunk 9]
```diff
@@ -881,6 +899,24 @@ \section*{Discussion}
...
+results that also endorse previous findings and lay insight to
+potential gut-microbial connections between cognition and neuroanatomy.
+For example, a longitudinal neurocognitive development study pointed at bilateral
+associations between the thalamus and the EL component of the Mullen scales.
+Linking to this observation, we found previously-mentioned taxa such as 
+\textit{B. wexlerae} and \textit{A. hadrus} to be among the top 15\% important predictors
+on both left and right thalamus and the EL subscale.
+Another instance of those findings is a previously reported association between
+the central opercular cortex (COC) and Gross Motor development. In this case, we identified that
+butyrate-producing \textit{Fusicatenibacter saccharivorans} is among the 15\% most important predictors
```
Comment (kescobo) [inline manuscript/main.tex:912]:
```suggestion
Similarly, a previously reported association between
the central opercular cortex (COC) and Gross Motor development % {{CITE}}
was mirrored in the microbiome in this cohort; we found that the
butyrate-producing \textit{Fusicatenibacter saccharivorans} is among the 15\% most important predictors
```

**Review by kescobo — CHANGES_REQUESTED** [review]
This is looking prety good!

File: manuscript/main.tex (hunk 9 above)
Comment (Hugemiler) [reply manuscript/main.tex]: Citation added on later commit

**Pull request merged**